src/core/search/compressed_sorted_set.cc

```diff
@@ -195,6 +199,33 @@ size_t CompressedSortedSet::ByteSize() const {
   return diffs_.size();
 }
 
+void CompressedSortedSet::Merge(CompressedSortedSet&& other) {
+  for (int v : other)
+    Insert(v);
```

Comment:
It looks like complexity can be reduced

Reply:
Same here, in theory yes, in practice it's linear